test-core/src/main/java/io/micronaut/test/extensions/AbstractMicronautExtension.java

```diff
@@ -74,7 +74,12 @@ public abstract class AbstractMicronautExtension<C> implements TestExecutionList
     public static final String TEST_TRANSACTIONAL = "micronaut.test.transactional";
     public static final String TEST_TRANSACTION_MODE = "micronaut.test.transaction-mode";
     public static final String DISABLED_MESSAGE = "Test is not bean. Either the test does not satisfy requirements defined by @Requires or annotation processing is not enabled. If the latter ensure annotation processing is enabled in your IDE.";
-    public static final String MISCONFIGURED_MESSAGE = "@MicronautTest used on test but no bean definition for the test present. This error indicates a misconfigured build or IDE. Please add the 'micronaut-inject-java' annotation processor to your test processor path (for Java this is the testAnnotationProcessor scope, for Kotlin kaptTest and for Groovy testCompile). See the documentation for reference: https://micronaut-projects.github.io/micronaut-test/latest/guide/";
+    public static final String MISCONFIGURED_MESSAGE = """
+        @MicronautTest used on test but no bean definition for the test present. \
+        This error indicates a misconfigured build or IDE. \
+        Please check 'micronaut.processing.group' or 'micronaut.processing.annotations' in your build. \
```

Comment:
what is meant by `Please check 'micronaut.processing.group' or 'micronaut.processing.annotations' in your build.`?

Reply:
In [the original issue](https://github.com/micronaut-projects/micronaut-test/issues/894), an issue was that the 

```
micronaut {
  processing {
    annotations('xxxx')
 }
}
```

Was incorrect, this is what I meant my `'micronaut.processing.annotations' in your build`

In Maven builds, this is a property `micronaut.processing.group`

Reply:
but that it is not required. is it? I mean you don't even have to use our build plugins to use Micronaut Test. am I missing something?